frontend/src/app/features/in-app-notifications/center/state/ian-center.service.ts

```diff
@@ -216,6 +216,17 @@ export class IanCenterService extends UntilDestroyedMixin {
     this.onReload.pipe(take(1)).subscribe((collection) => {
       this.store.update({ activeCollection: collection });
     });
+    if (facet === 'unread') {
+      this
+        .notifications$
+        .pipe(
+          take(1),
+        ).subscribe((notifications:INotification[][]) => {
```

Comment:
I am wondering whether it is really necessary to add a new subscription **every** time that we switch the facet.. That seems a bit overloaded to me.